openTSNE/nearest_neighbors.py

```diff
@@ -417,7 +418,9 @@ class NNDescent(KNNIndex):
         "yule",
     ]
 
-    def __init__(self, *args, **kwargs):
+    def __init__(
```

Comment:
Similarly to the Affinity classes, we'd want the option to pass in `knn_kwargs` to any KNN index. Here, I think you've missed the HNSW index and scikit-learn index.

Reply:
Passed is to the `Index` class for HNSW and to `NearestNeighbors` for sklearn